test/integration/targets/aws_kms/tasks/main.yml

```diff
@@ -108,14 +108,9 @@
       no_log: True
 
     - name: get ARN of calling user
-      command: "{{ ansible_python_interpreter }} -c 'import boto3, json; sts = boto3.client(\"sts\"); print json.dumps(sts.get_caller_identity())'"
-      changed_when: False
+      aws_caller_info:
```

Comment:
aws_caller_info accepts the same aws_access_key/aws_secret_key/security_token options that the other AWS modules accept, no need to build a hacky aws_environment fact any more